dask_jobqueue/htcondor.py

```diff
@@ -94,6 +96,24 @@ def __init__(
         if self.job_extra:
             self.job_header_dict.update(self.job_extra)
 
+        if submit_command_extra is None:
+            submit_command_extra = dask.config.get(
+                "jobqueue.%s.submit-command-extra" % self.config_name, []
+            )
+
+        self.submit_command = (
+            HTCondorJob.submit_command + " " + shell_quote_list(submit_command_extra)
+        )
+
+        if cancel_command_extra is None:
+            cancel_command_extra = dask.config.get(
+                "jobqueue.%s.cancel-command-extra" % self.config_name, []
+            )
+
+        self.cancel_command = (
+            HTCondorJob.cancel_command + " " + shell_quote_list(cancel_command_extra)
```

Comment:
not so sure if I see the benefit over directly calling `" ".join(shlex.quote(arg) for arg in cancel_command_extra)`